### PR TITLE
III-6438 Add new phpstan rule to prevent ramsey namespace

### PR DESCRIPTION
**Pull request by grubolsch**
### Added

To enforce the refactor the use our own Uuid implementation instead of the Ramsey library provided one, I added a custom phpstan rule to verify that Ramsey is never imported.

This is the final PR for this ticket.
---

Ticket: https://jira.uitdatabank.be/browse/III-6438

### CODE REVIEW

File: phpstan/ForbiddenRamseyNamespaceRule.php [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+    public function processNode(Node $node, Scope $scope): array
+    {
+        foreach ($node->uses as $use) {
+            if (! str_contains($use->name->toString(), 'Ramsey')) {
+                continue;
+            }
+
+            $filePath = $scope->getFile();
+            if (str_ends_with($filePath, 'src/Model/ValueObject/Identity/Uuid.php')) {
```
Comment (LucWollants) [inline phpstan/ForbiddenRamseyNamespaceRule.php:27]: If the Uuid file is moved this will break. Is it not possible to add and ignore comment?

Comment (grubolsch) [reply phpstan/ForbiddenRamseyNamespaceRule.php:27]: good idea, lets see...

Comment (grubolsch) [reply phpstan/ForbiddenRamseyNamespaceRule.php]: it works!

File: app/Phpstan/ForbiddenRamseyNamespaceRule.php [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        }
+
+        foreach ($node->uses as $use) {
+            if (! str_contains($use->name->toString(), 'Ramsey')) {
+                continue;
+            }
+
+            return [
+                sprintf(
+                    'The "Ramsey" namespace is not allowed in file: %s, please us CultuurNet\UDB3\Model\ValueObject\Identity\Uuid',
```
Comment (LucWollants) [inline app/Phpstan/ForbiddenRamseyNamespaceRule.php:32]: Nit: use `Uuid::class`?

**Pull request merged**